manifest/manifestlist/manifestlist_test.go

```diff
@@ -109,3 +110,105 @@ func TestManifestList(t *testing.T) {
 		}
 	}
 }
+
+// TODO (mikebrow): add annotations on the index and individual manifests
+var expectedOCIImageIndexSerialization = []byte(`{
```

Comment:
Does the serialization/deserialization work with an image index that references a manifest without a platform? The 'platform' property is optional per: https://github.com/opencontainers/image-spec/blob/master/image-index.md#image-index-property-descriptions

Sorry if this is obvious, but it is not immediately apparent to me (with my weak grasp of golang) :P